cfgmgr/coppmgr.cpp

```diff
@@ -0,0 +1,859 @@
+#include <fstream>
+#include "logger.h"
+#include "dbconnector.h"
+#include "producerstatetable.h"
+#include "tokenize.h"
+#include "ipprefix.h"
+#include "coppmgr.h"
+#include "exec.h"
+#include "shellcmd.h"
+#include "warm_restart.h"
+#include "json.hpp"
+
+using json = nlohmann::json;
+
+using namespace std;
+using namespace swss;
+
+static set<string> g_copp_init_set;
+static map<string, string> trap_feature_map = {
```

Comment:
In our case the trap ID disabled is samplepacket and feature associated is sflow. The key of the table shouldn't be considered here because the user has the option to change key and group different trap IDs.